app/assets/v2/css/gitcoin.css

```diff
@@ -1,7 +1,7 @@
 body {
   overflow-x: hidden;
   width: 100%;
-  background-color: #fff;
+  background-color: #EFF3F6;
```

Comment:
this file is loaded in many places and will change the bg color also there

Reply:
Thank you for the heads up great @octavioamu making changes asap!

Reply:
This stills need to be removed @molecula451

Reply:
This stills need to be removed @molecula451

Reply:
Did it